listenbrainz/webserver/views/playlist_api.py

```diff
@@ -76,24 +86,28 @@ def validate_playlist(jspf, require_title=True):
 def serialize_jspf(playlist: Playlist):
     """
         Given a playlist, return a properly formated dict that can be passed to jsonify.
-        TODO: Add tests for newly added fields.
-              Add collaborators
     """
 
     pl = {"creator": playlist.creator,
           "title": playlist.name,
           "identifier": PLAYLIST_URI_PREFIX + str(playlist.mbid),
-          "date": playlist.created.replace(tzinfo=datetime.timezone.utc).isoformat(),
+          "date": playlist.created.astimezone(datetime.timezone.utc).isoformat()
     }
     if playlist.description:
         pl["annotation"] = playlist.description
-    if playlist.created_for_id:
-        pl['created_for'] = playlist.created_for
 
-    extension = {"public": "true" if playlist.public else "false"}
-    extension["creator_id"] = playlist.creator_id
-    if playlist.copied_from_id:
-        extension['copied_from'] = PLAYLIST_URI_PREFIX + str(playlist.copied_from_id)
+    extension = {"public": playlist.public, "creator": playlist.creator}
+    if playlist.last_updated:
+        extension["last_modified_at"] = playlist.last_updated.astimezone(datetime.timezone.utc).isoformat()
+    if playlist.copied_from_id is not None:
+        if playlist.copied_from_mbid is None:
+            extension['copied_from_deleted'] = True
```

Comment:
we should document somewhere in the code (in addition to the mb.org/docs page) that we decided to serialise like this if a copied from playlist has been deleted

Reply:
Where would you suggest doing that?